iib/workers/tasks/build.py

```diff
@@ -1498,7 +1498,13 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     """
     package_name, labels = _apply_package_name_suffix(metadata_path, organization)
 
-    operator_manifest = OperatorManifest.from_directory(manifests_path)
+    try:
+        operator_manifest = OperatorManifest.from_directory(manifests_path)
+    except ruamel.yaml.YAMLError:
```

Comment:
Are you sure that just this error can be raised from `OperatorManifest.from_directory`

https://github.com/containerbuildsystem/operator-manifest/blob/master/operator_manifest/operator.py#L579

Shouldn't we just catch all the exception and properly log and return them?

Reply:
@MartinBasti  @yashvardhannanavati  I added (RuntimeError, ValueError) , let me know if you want to catch also general exception. Unit tests added.